QuickViewFile/Models/ConfigModel.cs

```diff
@@ -15,7 +23,21 @@ public class ConfigModel
         public double MaxScale { get; set; } = 100.0;
         public double MinScale { get; set; } = 1;
         public double MouseWheelZoomStepFactor { get; set; } = 1.2;
-        public string BitmapScalingMode { get; set; } = "Fant";
+
+        private string _bitmapScalingMode = "Fant";
```

Comment:
@jules Why you're chanigng this and doing so by property change when... you never saves this anywhere else. It will only be stored until user closes application...